ext/Images/FeatureMountain.jl

```diff
@@ -396,13 +375,17 @@ end
 
 
 function unionFeatureMountain(
-  fMa::Dict{Tuple{Symbol,Int},MANYTRACKS}, 
-  fMb::Dict{Tuple{Symbol,Int},MANYTRACKS},
+  fMa::FEATUREMOUNTAINTYPES, # Dict{Tuple{Symbol,Int},MANYTRACKS}, 
+  fMb::FEATUREMOUNTAINTYPES, # Dict{Tuple{Symbol,Int},MANYTRACKS},
 )
   # start with everything from fMb
+  # FIXME, this deepcopy is going to explode the memory consumption
   rM = deepcopy(fMb)
```

Comment:
HI @Affie , this may be part of the memory use issue.  This PR will help some, and we can refactor more in a follow-up PR.